client/Annotators/EDF/timeSeriesAnnotator.js

```diff
@@ -3931,6 +3931,8 @@ $.widget("crowdeeg.TimeSeriesAnnotator", {
         if(that.options.context.preferences.uploadedPreferences.scalingFactors != null){
           if(Object.keys(that.options.context.preferences.uploadedPreferences.scalingFactors).length != Object.keys(that.vars.originalScalingFactors).length){
             window.alert("The preferences file you uploaded is not compatible with the chart (number of channels do not match). Please choose another file.");
+          } else if (that.options.context.preferences.uploadedPreferences.startTime < 0 || that.options.context.preferences.uploadedPreferences.startTime > that.vars.recordingLengthInSeconds) {
```

Comment:
This might be better to display when the user tries to upload the file (i.e. in _parsePreferencesJsonFile()) instead of when they try to save it, just to warn the user earlier.